src/index.ts

```diff
@@ -13,17 +13,20 @@ export interface ICred {
   sessionToken: string | undefined
 }
 
+export interface EthAddress {
```

Comment:
❔ Would it be useful to include a docstring indicating what this interface is used for?